llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -1737,8 +1738,13 @@ PassBuilder::buildPerModuleDefaultPipeline(OptimizationLevel Level,
   // Emit annotation remarks.
   addAnnotationRemarksPass(MPM);
 
+  // Count the types of instructions used
```

Comment:
I told you to put this here but actually can you put it below the `isLTOPrelink` part below to ensure this runs after all other passes?